gradle/libs.versions.toml

```diff
@@ -8,7 +8,7 @@ kotlin = '2.0.20'
 hilt = '2.47'
 jacoco = '0.8.10'
 designSystem = "0.4.1-SNAPSHOT"
-dhis2sdk = "1.11.1-SNAPSHOT"
+dhis2sdk = "1.11.1-20241218.080719-10"
```

Comment:
Should we need to wait until this is in the snapshot version?

Reply:
yes we are going to wait before merge